site2/docs/security-kerberos.md

```diff
@@ -21,7 +21,9 @@ If your organization is already using a Kerberos server (for example, by using `
 
 Note that if you are using Oracle Java, you need to download JCE policy files for your Java version and copy them to the `$JAVA_HOME/jre/lib/security` directory.
 
-#### Kerberos Principals
+Here is some other prerequisites that will use in this configuration.
+
+#### 1.1.1. Kerberos Principals
```

Comment:
You want to use the ordered number list to show the hierarchy and structure cleary, that's good, but a more effective way is to generate the ordered number list automatically rather than manually.